setup.py

```diff
@@ -43,7 +43,8 @@
         "pandas>=1.0.0,<3.0.0",
         "bluepysnap>=1.0.5,<2.0.0",
         "pydantic>=1.10.2,<2.0.0",
-        "typing-extensions>=4.8.0"
+        "typing-extensions>=4.8.0",
+        "networkx"
```

Comment:
Could you add the upperbound for networkx as well?
`networkx>=3.2.1` or `networkx>=3.2.0`?